README.md

```diff
@@ -0,0 +1,72 @@
+# fstab
+
+This Role generates a proper apt sources.
+
+
+## Requirements
+
+A running debian based system.
+
+
+## Role Variables
+
+| Name                                    | Type                                 | Description                                                                                                                                                                       |
+|-----------------------------------------|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `apt_sources_ubuntu`                    | [dict](#apt_sources_ubuntu) | Apt source config for ubuntu systems                                                                                                                                              |
+| `apt_sources_debian`                    | [dict](#apt_sources_debian) | Apt source config for debian systems                                                                                                                                              |
+| `apt_sources_additional_source_strings` | list of strings                      | Additional apt sources to add (each string is additional line in `/etc/apt/sources.list` without any modifications)(This option will be overwritten by the release specific ones) |
```

Comment:
This variable is never used. If you do decide to implement this, please mention how it will be overridden. Will an empty list in the Ubuntu dict override it, or will it only override it if the list has items?